src/evaluators/utility/PHAL_Field_Source_Def.hpp

```diff
@@ -95,11 +95,12 @@ namespace PHAL
                     Teuchos::RCP<Teuchos::ParameterList> p = Teuchos::rcp(new Teuchos::ParameterList(Albany::strint("Gaussian: Amplitude", num)));
                     p->set<Teuchos::RCP<ParamLib>>("Parameter Library", paramLib);
                     p->set<std::string>("Parameter Name", param_name);
+                    p->set<Teuchos::RCP<Albany::AccessorsMap>>("Accessors", scalarParam_list.get<Teuchos::RCP<Albany::AccessorsMap>>("Accessors"));
```

Comment:
Another point in support of not creating accessors in SharedParam: it seems silly to pass around the whole map when we know already which accessor will be needed. If only they were already created outside (by the problem), we'd be able to pass around only the one that is needed.